server/app/components/product-header.tsx

```diff
@@ -0,0 +1,33 @@
+import { Card } from "@tremor/react";
+
+interface ProductHeaderProps {
+  product: {
+    image_url: string;
+    name: string;
+  };
+}
+
+const ProductHeader = ({ product }: ProductHeaderProps) => (
+  <>
+    {product?.image_url && (
+      <Card className="mx-auto grid w-full grid-cols-1 gap-1 space-y-4 md:h-full md:grid-cols-7 md:gap-6 lg:w-3/4">
+        <span className="grid justify-center">
+          <img
+            className="my-auto"
+            src={product.image_url.replace("US120", "US240")}
```

Comment:
Is there a benefit to using US240 if the width is fixed to 120px anyway?

Reply:
yea, it looks better. browser pixels aren't real pixels